### PR TITLE
Style/chat

### PR DESCRIPTION
**Pull request by danielcgilibert**
# Description

This PR changes the design of the chat and improves it on mobile.

## Checklist before requesting a review

Please delete options that are not relevant.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code

## Screenshots (if appropriate):

before:
SCR-20230607-hhe
SCR-20230607-hi1

After:
SCR-20230607-hcr
SCR-20230607-hdg
SCR-20230607-hdr

### CODE REVIEW

**Comment by StanGirard** [comment]
@iMADi-ARCH and @mamadoudicko my frontend Kong what do you think ?

**Comment by StanGirard** [comment]
Thanks a lot for the PR ! I'll check it very soon

**Comment by StanGirard** [comment]
@danielcgilibert 

image


Run `yarn build` locally to make sure it works ;)

File: frontend/app/chat/page.tsx [hunk 1]
```diff
@@ -1,85 +1,100 @@
...
+            <div className="flex gap-1 w-full">
+              <input
+                autoFocus
+                value={question}
+                type="text"
+                onChange={e => setQuestion(e.target.value)}
+                onKeyDown={e => {
+                  if (e.key === "Enter" && !e.shiftKey) {
+                    e.preventDefault() // Prevents the newline from being entered in the textarea
+                    if (!isPending) askQuestion() // Call the submit function here
```
Comment (mamadoudicko) [inline frontend/app/chat/page.tsx]: Since the input is no longer a `<textarea/>`, the condition `!e.shiftKey` seems deprecated

**Review by mamadoudicko — CHANGES_REQUESTED** [review]
Nice PR, @danielcgilibert! I really like the new chat UI 😍

There seems to be an issue with your commits. The second one reverts some changes from the first commit.

For example, in your first commit, you removed `style={{ marginBottom: '20px' }}`, but the second commit reintroduced it back.

The third commit can also be merged into the first one since that's where `left` notion was removed.

Could you please try to group the three commits into one? [You can refer to this article for guidance.](https://medium.com/@igor_marques/combining-two-commits-84281f470ee8)

**Comment by StanGirard** [comment]
@mamadoudicko no sure if relevant the comment on the git history because I squash all of them into one when merging ;)

**Comment by mamadoudicko** [comment]
We'll still lose the changes from the first commit since the second one undoes them

**Pull request merged**